src/@types/bs58/index.d.ts

```diff
@@ -2,3 +2,5 @@ declare module 'bs58' {
   export function encode(data: any): any;
   export function decode(data: any): any;
 }
+
+declare module 'blake-hash';
```

Comment:
src/@types directory is structured per file and I don't think `blake-hash` is part of `bs58` lib. Please put it on its own file.

Reply:
Thanks. Done. Merge.